### PR TITLE
Install packages as user

### PR DESCRIPTION
**Pull request by pivnicek**
PS-24 fix, requires https://github.com/keboola/python-transformation/pull/9

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -26,4 +26,4 @@ services:
     environment:
       - PASSWORD=test
       - TAGS=[]
-      - PACKAGES=[]
+      - PACKAGES=["scipy", "numpy"]
```
Comment (odinuv) [inline docker-compose.yml]: can we make this just `PACKAGES` ans use .env file?

File: Dockerfile [hunk 2]
```diff
@@ -136,7 +136,7 @@ RUN MPLBACKEND=Agg python -c "import matplotlib.pyplot"
 
 ### Custom stuff
 # Install KBC Transformation package
-RUN pip3 install --no-cache-dir --upgrade git+git://github.com/keboola/python-transformation.git@1.1.13
+RUN pip3 install --no-cache-dir --upgrade git+git://github.com/keboola/python-transformation.git@PS-24-user-option
```
Comment (odinuv) [inline Dockerfile]: don't forget to put this to release

**Review by odinuv — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (pivnicek) [reply docker-compose.yml]: sure

File: Dockerfile (hunk 2 above)
Comment (pivnicek) [reply Dockerfile]: jasne ;-)

**Comment by pivnicek** [comment]
@odinuv I updated it, could you please give it a sanity check?

**Review by odinuv — APPROVED** [review]
✅

**Pull request merged**